app/components/featured_blog_posts_component.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class FeaturedBlogPostsComponent < ViewComponent::Base
+  def initialize(number_to_display:, show_main_feature: true)
+    @show_main_feature = show_main_feature
```

Comment:
It doesn't appear that we're using this in the erb template, so could this assignment be removed?